src/options.asm

```diff
@@ -97,20 +94,30 @@ BANK_1C_VAR = 1
 ;   - clamping the maximum health between 3 and 14 before loading a saved file
 ;   - recalculating the maximum health to the actual, legal amount before saving,
 ;     based on map room flags for defeated bosses and collected heart containers
+;
+; __OPTIMIZATIONS_1__
+;   Optimizations that were implemented in version 1.1 of the Japanese and English
+;   revisions, and also in 1.0 of the French and German releases
+;
+; __OPTIMIZATIONS_2__
+;   Small optimizations that were likely bundled in when adding diacritics support,
+;   and so appear in the Japanese and German versions, but not any others
```

Comment:
Maybe also a stub comment mentioning OPTIMIZATIONS_3  ?

Reply:
Of course!